plan/logical_plan_test.go

```diff
@@ -800,7 +804,7 @@ func (s *testPlanSuite) TestRefine(c *C) {
 		},
 		{
 			sql:  "select b from t where c = 1 or c = 2 or c = 3 or c = 4 or c = 5",
-			best: "Index(t.c_d_e)[[1,1] [2,2] [3,3] [4,4] [5,5]]->Projection",
+			best: "Table(t)->Projection",
```

Comment:
Then you'd better split the logic for estimating with pseduo table and real table, because we'd better use index as much as possible if we don't have a stastic info. It's the best practice.
If you want to test, you should insert some data and analysis it.